src/oneD/OneDim.cpp

```diff
@@ -338,33 +337,44 @@ double OneDim::timeStep(int nsteps, double dt, double* x, double* r, int logleve
     // set the Jacobian age parameter to the transient value
     newton().setOptions(m_ts_jac_age);
 
-    debuglog("\n\n step    size (s)    log10(ss) \n", loglevel);
-    debuglog("===============================\n", loglevel);
-
     int n = 0;
     int successiveFailures = 0;
 
+    // Only output this if nothing else under this function call will be output
+    if (loglevel == 1){
+        debuglog("\n============================", loglevel);
+        debuglog(fmt::format("\n{:<5s}  {:<11s}   {:<7s}\n", "step", "dt (s)", "log(ss)"), loglevel);
+        debuglog("============================", loglevel);
+    }
```

Comment:
You don't need to use `debuglog` if you're already using an `if (loglevel)` conditional, in which case you can also elide the explicit call to `fmt::format`.

Also, as a minor formatting nit, there should be a space between the `)` and the `{` in the `if` statement.